LDK/test/src/org/labkey/test/util/external/labModules/LabModuleHelper.java

```diff
@@ -50,9 +51,21 @@
     public static final String NEG_COLOR = "rgb(255, 255, 0)";
     public static final String STD_COLOR = "rgb(0, 128, 0)";
 
+    public static final int VIRAL_BATCH_FIELDS = 0;
+    public static final int VIRAL_RUN_FIELDS = 1;
+    public static final int VIRAL_RESULT_FIELDS = 2;
+
     public LabModuleHelper(BaseWebDriverTest test)
     {
         _test = test;
+
```

Comment:
I'm confused - why is this in the constructor?

Reply:
Confusing indeed - this was merge gone bad, now remove. Good catch!